osmaxx/conversion/converters/converter_gis/bootstrap/sql/functions/0060_cast_to_float_or_null.sql

```diff
@@ -0,0 +1,10 @@
+-- try casting to float and return NULL if failed
+create or replace function cast_to_float_null_if_failed(v_input text) returns float as $$
+begin
+    return cast(v_input as float);
+exception
+    when invalid_text_representation then
+        RAISE NOTICE 'Invalid float value: "%".  Returning NULL.', v_input;
+        return NULL;
+end;
+$$ language plpgsql immutable;
```

Comment:
I know our current scripts have varying mixtures, but please be consistent at least for new scripts: Please either write all SQL and PL/SQL keywords in all-caps or write all of them all-lowercase.